gradle/versions.gradle

```diff
@@ -9,7 +9,7 @@ ext.versions = [
 	'xtext_bootstrap': '2.14.0',
 	'gradle_plugins': '0.1.0',
 	'xtext_gradle_plugin': '1.0.21',
-	'lsp4j': '0.4.0', // dont forget to adapt IdeProjectDescriptor + Tests
+	'lsp4j': '0.5.0-SNAPSHOT', // dont forget to adapt IdeProjectDescriptor + Tests
```

Comment:
miss adaptation of IdeProjectDescriptor and tests. maybe we can use the snapshot version in the wizard if the Xtext version is a snapshot and the release version if it is a release

Reply:
Thanks for the note, correct. As I stated in the PR description, I need to some input on how to deal with the different LSP4J versions.